infrateam_first_responder.md

```diff
@@ -48,34 +48,35 @@ The first responder needs to make sure that all codebases needing updates have u
 
 Run the `merge-all` script to automatically merge all dependency update PRs: https://github.com/sul-dlss/access-update-scripts/blob/master/merge-all.rb. Note that this script will only work with Ruby 2.6 or greater.  See the comments at the top for how to run and note you will need a github access token if you haven't previously created one.  Instructions for creating a token are here:  https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line   Save your token somewhere secure for re-use since you won't be able to view it in the Github interface again.
 
-### Run infrastructure-integration-tests
+### Deploy 'em
 
-We want the FR to ensure
-  - dependency updates don't break cross-app functionality
-  - this test suite remains useful
+Use the `sdr-deploy` CLI to deploy all infrastructure projects (with **required additional deploys** noted below) via capistrano to deployed environments: https://github.com/sul-dlss/sdr-deploy
 
-To that end, all the infrastructure-integration-tests must be run successfully in the qa and/or stage environment before deploying to production (one of qa or stage is sufficient;  testing in environments helps ensure our environments are (still) set up properly.).  This should be done as part of deployment of dependency updates as described in the next section.
+Note that you will need to be sure you can ssh into each of the VMs from wherever you are running the `sdr-deploy` CLI. (See the [sdr-deploy README](https://github.com/sul-dlss/sdr-deploy/blob/main/README.md) for more about how to do this.)
 
-If some tests fail when running the whole test suite at once, but pass when run individually, that is ok -- as long as they each pass under some circumstances. Chances are not great that the full suite will pass in one go from a spotty off-campus connection.
+#### 1. Create a release tag
 
-If you're unsure whether a particular test failure indicates bad network luck, a regression in the application, or an out of date test, raise it for discussion (or ask another dev to retry from their laptop) in #dlss-infrastructure.
+First, use `sdr-deploy` to create a release tag. This lets you deploy a known point in time without, e.g., asking other developers to hold commits while deployments are in process. It also lets us rollback to a known good tag. (See the [sdr-deploy README](https://github.com/sul-dlss/sdr-deploy/blob/main/README.md) for more about how to do this.)
```

Comment:
Second sentence awkward.

Reply:
Works for me.

Reply:
hold merges to `main` ?

Reply:
@ndushay tweaked the suggestion above, thanks! I'll patch it in now.